program-runtime/src/compute_budget.rs

```diff
@@ -175,6 +177,7 @@ impl ComputeBudget {
         support_request_units_deprecated: bool,
         enable_request_heap_frame_ix: bool,
         support_set_loaded_accounts_data_size_limit_ix: bool,
+        round_compute_unit_price: bool,
```

Comment:
This flag is added only for testing purpose?

Reply:
it's for the feature-gate